app.py

```diff
@@ -1023,14 +1050,36 @@ def handle_option(self, option):
                     print("Invalid choice. Try again.")
                     update = False
 
-        # Exit
+        # Load Product File
         elif option == '3':
+            clear = True
+
+            while True:
+                # Create load product file Menu
+                if update:
+                    self.display_menu(MenuType.LOAD_PRODUCT_FILE, clear=clear)
+                    clear = True
+                else:
+                    update = True
+                
+                suboption = input("> ")
+
+                # Set Product File Name
+                if (suboption == '1'):

```

Comment:
Remove the load product submenu, if we choose the `Load Product` option from the Settings Menu, it should directly ask the user to enter the file location.